src/utils/duration-humanizer.ts

```diff
@@ -0,0 +1,39 @@
+import * as humanizeDuration from 'humanize-duration';
+
+const getTimeInText = (position: number, uiConfig: any) => {
```

Comment:
add a TODO about exporting to common, unless this is a different usecase than related

Reply:
the use-case is the same but it's a bit different in related from what i saw; anyway i think it will be a good idea to export to a common place.